install-i3lock-color.sh

```diff
@@ -6,3 +6,11 @@ cd build
 sudo make install
 
 sudo install -Dm644 ../LICENSE "/usr/share/licenses/i3lock-color/LICENSE"
+
+if [[ -f /usr/bin/zsh ]]; then
+  sudo cp ../_i3lock /usr/share/zsh/vendor-completions
+fi
+
+if [[ -f /usr/bin/bash ]]; then
+  sudo cp ../i3lock-bash /usr/share/bash-completion/completions/i3lock
+fi
```

Comment:
I don't actually know how all the make process works here, so I made this workaround

Reply:
Make uses m4 (GNU autotools and automake). Yes it's very annoying. I'm going to integrate this with m4 before I merge it. Thanks again for contributing :slightly_smiling_face:

Reply:
Well, I played around with autotools/automake and moved the completions to it :D

Reply:
Nice. Thanks for that :)